requests/models.py

```diff
@@ -776,23 +776,43 @@ def iter_lines(self, chunk_size=ITER_CHUNK_SIZE, decode_unicode=None, delimiter=
 
         .. note:: This method is not reentrant safe.
         """
-
         pending = None
 
-        for chunk in self.iter_content(chunk_size=chunk_size, decode_unicode=decode_unicode):
+        for chunk in self.iter_content(chunk_size=chunk_size,
+                                       decode_unicode=decode_unicode):
+            # Skip any null responses
+            if not chunk:
+                continue
```

Comment:
You're right
Fixed and added tests for this behavior :ok_hand:

I don't know if it's licit to receive empty chunks in the middle of a stream, but I guess it's possible in theory. (and it would work)